tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  params:
+    - name: mrrcParamFilePath
+      description: path of the env file for mrrc parameters to use
+      type: string
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+      type: string
+  steps:
+    - name: prepare-repo
+      image: quay.io/konflux-ci/release-service-utils:4576e1e2a30439129cb69c8a4f39f7ced2d44376
+      script: |
+        set -eux
+
+        MRRC_FILE="$(workspaces.data.path)/$(params.mrrcParamFilePath)"
+        . "$MRRC_FILE"
+        echo "Downloading the maven repo zip $MRRC_ZIP_REGISTRY"
+        mrrc_dir="$(dirname $MRRC_FILE)/mrrc"
+        mkdir -p $mrrc_dir
+        cd $mrrc_dir
+        oras pull $MRRC_ZIP_REGISTRY
+    - name: upload-maven-repo
+      image: quay.io/konflux-ci/charon:b99585182f6750b3f6352889fe9c2222b57833ef
```

Comment:
Now that this image has been made public, I wonder if it would make sense to simply include charon in the utils image, like we include e.g. oras: https://github.com/konflux-ci/release-service-utils/blob/main/Dockerfile#L19

Or maybe that wouldn't be as simple as that as there might be more things and not just one binary?

Just wanted to ask what you think of this. Using the charon image should be fine as well, especially since it's hosted in the quay.io/konflux-ci namespace.

Reply:
The charon tool is developed using python which has lots of dependencies, so it's a little hard to include it in release-service-utils image. This is the Dockerfile for it: https://github.com/Commonjava/charon/blob/konflux/image/Containerfile
I think if we want to include it, we need to add bunch of changes in the release-service-utils. And there is an undetermined requirement which may need some other tools to do "detached signatures"  in next version. I'm not sure any new changes to the image, so maybe we should leave it single now.

Reply:
I am fine with it not being in our image. However, we don't like to have images that we don't "control" because if someone goes and deletes it, our pipelines break and we are on the hook for it. I am fine with this being merged like this, but can you file a ticket on the RELEASE Jira board for us to copy the image listed here to one of our quay orgs and update this line?

Reply:
Ok, I'll do this. Do we have a template or sample ticket for me to reference?

Reply:
> The charon tool is developed using python which has lots of dependencies
Yeah, then I agree it makes sense to use a separate image.

Reply:
The ticket is created for tracking.

Reply:
Here's the ticket for reference: https://issues.redhat.com/browse/RELEASE-1154